packages/plugins/integration-make-com-ui/src/lib/components/make-com-authorize/make-com-authorize.component.html

```
@@ -4,52 +4,23 @@
-            {{ 'BUTTONS.NEXT' | translate }}
-          </button>
-        </form>
-      </nb-card-body>
-    </nb-card>
+    <div class="col-xl-6 col-12">
+      <div class="hint mb-4">
+        <nb-icon icon="info-outline"></nb-icon>
+        {{ 'INTEGRATIONS.MAKE_COM_PAGE.CONNECT_INFO' | translate }}
+      </div>
+      <button
+        nbButton
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)